flake.nix

```
@@ -157,8 +157,9 @@
       common-gpu-amd-sea-islands = import ./common/gpu/amd/sea-islands;
       common-gpu-amd-southern-islands = import ./common/gpu/amd/southern-islands;
       common-gpu-intel = import ./common/gpu/intel.nix;
-      common-gpu-nvidia = import ./common/gpu/nvidia.nix;
-      common-gpu-nvidia-disable = import ./common/gpu/nvidia-disable.nix;
+      common-gpu-nvidia = import ./common/gpu/nvidia/prime.nix;
+      common-gpu-nvidia-nonprime = import ./common/gpu/nvidia;
+      common-gpu-nvidia-disable = import ./common/gpu/nvidia/disable.nix;
```

Could we reverse this to keep backwards compatibility?
```suggestion
      common-gpu-nvidia = import ./common/gpu/nvidia;
      common-gpu-nvidia-nonprime = import ./common/gpu/nvidia/prime.nix;
      common-gpu-nvidia-disable = import ./common/gpu/nvidia/disable.nix;
```